geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitor.java

```diff
@@ -130,7 +130,7 @@
   public static final long MEMBER_SUSPECT_COLLECTION_INTERVAL =
       Long.getLong("geode.suspect-member-collection-interval", 200);
 
-  private volatile long currentTimeStamp;
+  volatile long currentTimeStamp;
```

Comment:
Should we maybe remove direct access to this field and rather replace it with an accessor that we can then mark with `@VisibleForTesting`?

Reply:
seems the same as membersInFinalCheck to me, allowing a test subclass to access it